### PR TITLE
support another job queue  python-rq

### PR DESCRIPTION
**Pull request by yxy**
I don’t use celery current now, instead I use python-rq which is simpler  and lighter than Celery. 

So, I added this patch , hope someone like me using rq as well will take advantage of it.

### CODE REVIEW

File: django_slack/tasks.py [hunk 1]
```diff
@@ -1,9 +1,18 @@
-from celery import task
-
 from django.utils.module_loading import import_string
 
 from .app_settings import app_settings
 
-@task
-def send(*args, **kwargs):
```
Comment (clokep) [inline django_slack/tasks.py]: I think this change will cause auto-registering of tasks to break in celery.

**Comment by yxy** [comment]
@clokep  I added some tests to make sure we get the task instance be the CeleryTask. 

do you want me open another PR.

and sorry, this pull request can be broken, due to I forget to commit the other file.
Please close this PR.

File: django_slack/backends.py [hunk 2]
```diff
@@ -60,4 +59,24 @@ def send(self, *args, **kwargs):
...
+
+    def init_sender(self):
+        from .tasks import get_celery_task
+        return get_celery_task()
+
+
+class RQBackend(_AsyncBackend):
+
+    def init_sender(self):
+        from .tasks import get_rq_worker
```
Comment (yxy) [inline django_slack/backends.py]: this get_rq_worker function can never be imported !!! :(

File: django_slack/tasks.py (hunk 1 above)
Comment (yxy) [inline django_slack/tasks.py]: I will set some test to test whether it can be work. sorry for the untest code.

Comment (yxy) [inline django_slack/tasks.py]: Yes, you are right, due to celery running in another process, tasks need to be import before executing. 
So, I guess I have to separate these two modules.

I manage to fixed it. 
So, do you want me open another PR? or just leave you to do this。

**Comment by lamby** [comment]
Closing by request
